### DISCUSSION TITLE
Tip: you can save the Volume Mixer's window position with FancyZones in Microsoft PowerToys

### DISCUSSION THREAD

**Discussion by TwoCables** [File-New-Project/EarTrumpet#1364] [Completed discussions]
First, install PowerToys and make sure its in-app setting called "Run at startup" is on.

Then go to FancyZones. From there, launch the FancyZones layout editor. Then you just set up your zones so that you have a zone that works for your preferred location of the Volume Mixer (it will snap to the top-left corner of any zone). Then you hold Shift while you drag the Volume Mixer to that zone and then release it. When you release it, it snaps into that zone. So you hold down Shift while clicking and dragging, and then you release when it's in the zone you want it to be in and it snaps in place.

From that point forward, Volume Mixer will always open to that exact location (when you open it), but only if PowerToys is running (you'll see its icon in the System Tray). Just a word of caution though: if you ever move a window that's snapped to a zone by even 1 pixel using just a normal Click & Drag (so, without holding down Shift), then it will get unsnapped from its zone, which means you'll have to snap it back to that zone again.

So yeah, get fancy and get your Volume Mixer in the zone! I couldn't resist.

**Edit:** This applies to Ear Trumpet's settings as well and there's no way to separate them into two different zones.

**Comment by RensDuijsens** [comment]
Yes....that works......but the volume mixer does not start automatically when Windows starts.

**Comment by TwoCables** [comment]
Settings > Community > Show full mixer window on startup

**Comment by riverar** [comment]
This is already supported in our dev builds. (Remembering the last position.) Please let us know if you have issues with it.
